include/sys/device.h

```diff
@@ -26,6 +26,7 @@ typedef int (*d_detach_t)(device_t *dev);
 typedef enum {
   DIF_BUS,
   DIF_PCI_BUS,
+  DIF_ICU,
```

Comment:
could you please expand ICU acronym in comment?

Reply:
ok.

Reply:
Done.

Reply:
thanks